tests/scripts/run_pre_post_commit_regressions_fast_dispatch.sh

```diff
@@ -24,5 +24,3 @@ export PYTHONPATH=$TT_METAL_HOME
 
 ./tests/scripts/run_cpp_unit_tests.sh
 
-# build docs
-./tests/scripts/run_build_docs.sh
```

Comment:
is this script called anywhere now?

Reply:
I am trying to call it from the docs workflow but it does not seem to work without a lot of other modifications.

Reply:
I want to still give a way for devs to build the docs locally, is this the only way now?

Reply:
I think they can still run `./tests/scripts/run_build_docs.sh` to build docs. We do not have a tox step for docs - there are too many C++ dependencies to manage well in Tox. 

One thing I do want to do if I can is to make sure that locally and on CI the same script is executed:

i.e. locally devs call `./tests/scripts/run_build_docs.sh` and CI calls `./tests/scripts/run_build_docs.sh`

Reply:
yes good idea, otherwise it may be broken if we don't use it in CI